### ISSUE TITLE
[Bug] Wrong code generated in options mode

### ISSUE THREAD

**Issue by songzhi** [ferdikoomen/openapi-typescript-codegen#233]
A path takes 2 parameters `a` and `b`, `b` is not required and has a default value. 
Generated code by now:
```ts
function add({ a, b = 1 }: { a: number; b: number }): number {
  return a + b
}
```
Which is making `b`'s default value useless because we must give it a value to pass the type check.
Correct code should be:
```ts
function add({ a, b = 1 }: { a: number; b?: number }): number {
  return a + b
}
```
It has been tested, if call `add` like `add({a:1})`, `b` will set as it's default value.

**Comment by ferdikoomen** [comment]
Hi @songzhi there is a fix pushed in the master branch, will deploy a new version after some testing

**Comment by ferdikoomen** [comment]
just released version 0.2.9

**Comment by ferdikoomen** [comment]
Pushed version 0.3.1 that has some performance improvement due to precompiled templates.
